programmers/음양-더하기.js

```diff
@@ -0,0 +1,22 @@
+function solution(absolutes, signs) {
+    let result = 0;
+    
+    for(let i = 0; i < signs.length; i++) {
+      if (signs[i] === false) {
+        absolutes[i] = -absolutes[i];
+      }
+      result += absolutes[i];    
+    }
+    return result;
+}
+// for문 풀이입니다 
+
+
+function solution(absolutes, signs) {
+    for(let i = 0; i < signs.length; i++) {
+      if (signs[i] === false) absolutes[i] = -absolutes[i];
+    }
+    let result = absolutes.reduce((a, b) => a + b, 0 );
+    return result
+}
```

Comment:
1. for 다음 공백
2. 굳이 result 변수에 담아서 return 할 필요 없이 바로 return 때려도 됨

```suggestion
function solution(absolutes, signs) {
    for (let i = 0; i < signs.length; i++) {
      if (signs[i] === false) absolutes[i] = -absolutes[i];
    }
    return absolutes.reduce((a, b) => a + b);
}
```

Reply:
이렇게 forEach 메서드를 사용해서 해결할 수도 있음.

```js
function solution(absolutes, signs) {
    signs.forEach((sign, i) => {
        if (signs[i] === false) absolutes[i] = -absolutes[i]
    });
    
    return absolutes.reduce((a, b) => a + b);
}
```